celery/utils/serialization.py

```diff
@@ -80,6 +81,19 @@ def itermro(cls, stop):
 
 def create_exception_cls(name, module, parent=None):
     """Dynamically create an exception class."""
+    try:
+        # also works for 'builtins' module
+        mod = import_module(module)
```

Comment:
What happens when the module has side effects? You'll get a remote code execution vulnerability if you can inject a result somehow.

Reply:
If the module is being used in a task, it's already inside a trust boundary.  So some trust is assumed.

My head goes towards sandboxing the Exception inspection.  Like, run it in a locked down subprocess, just to get the signature.  I need to look at the code closer though.